desktop/release-notes.md

```diff
@@ -24,11 +24,85 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.19.0
+
+{% include release-date.html date="2023-04-27" %}
+
+{% include desktop-install.md all=true version="4.19.0" build_path="/" %}
+
+### New
+
+TODO: Work in progress
+
+### Removed
+
+- Removed `docker scan` command. To continue learning about the vulnerabilities of your images, and many other features, use the new `docker scout` command. Run `docker scout --help`, or [read the docs to learn more](../engine/reference/commandline/scout.md).
+
+### Upgrades
+
+- [Docker Engine v23.0.3](https://docs.docker.com/engine/release-notes/23.0/#2303)
+- [Compose 2.17.3](https://github.com/docker/compose/releases/tag/v2.17.3)
+- [Containerd v1.6.20](https://github.com/containerd/containerd/releases/tag/v1.6.20)
+- [Kubernetes v1.26.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.26.3)
+- [runc v1.1.5](https://github.com/opencontainers/runc/releases/tag/v1.1.5)
+- [criTools v1.26.1](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.26.1)
+- [cri-dockerd v0.3.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.3.1)
+- [coredns v1.9.3](https://github.com/coredns/coredns/releases/tag/v1.9.3)
+- [cni v1.2.0](https://github.com/containernetworking/plugins/releases/tag/v1.2.0)
+- [Go v1.20.3](https://github.com/golang/go/releases/tag/go1.20.3)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Improved `docker scout compare` command to compare two images, now also aliased under `docker scout diff`.
+- Added in-app walkthroughs to Learning Center.
+- Added more details to dashboard errors when a `docker-compose` action fails ([docker/for-win#13378](https://github.com/docker/for-win/issues/13378)).
+- Added support for setting HTTP proxy configuration during installation. This can be done via the `--proxy-http-mode`, `--overrider-proxy-http`, `--override-proxy-https` and `--override-proxy-exclude` installer flags in the case of installation from the CLI on [Mac](../install/mac-install.md#install-from-the-command-line) and [Windows](../install/windows-install.md#install-from-the-command-line), or alternatively by setting the values in the `install-settings.json` file.
```

Comment:
```suggestion
- Added support for setting HTTP proxy configuration during installation. This can be done via the `--proxy-http-mode`, `--overrider-proxy-http`, `--override-proxy-https` and `--override-proxy-exclude` installer flags in the case of installation from the CLI on [Mac](/install/mac-install.md#install-from-the-command-line) and [Windows](/install/windows-install.md#install-from-the-command-line), or alternatively by setting the values in the `install-settings.json` file.
```